java/client/src/main/java/glide/api/commands/HashCommands.java

```diff
@@ -0,0 +1,49 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Hash Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=hash">Hash Commands</a>
+ */
+public interface HashCommands {
+
+    /**
+     * Retrieve the value associated with <code>field</code> in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hget/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @param field The field in the hash stored at <code>key</code> to retrieve from the database.
+     * @return The value associated with <code>field</code>, or <code>null</code> when <code>field
+     *     </code> is not present in the hash or <code>key</code> does not exist.
+     */
+    CompletableFuture<String> hget(String key, String field);
+
+    /**
+     * Sets the specified fields to their respective values in the hash stored at <code>key</code>.
```

Comment:
We need to keep tense consistent (even if other clients are not)
```suggestion
     * Set the specified fields to their respective values in the hash stored at <code>key</code>.
```